.travis.yml

```diff
@@ -1,30 +1,17 @@
 language: ruby
-os: osx
-env: OSX=10.11
-osx_image: xcode7.3
-rvm: system
+rvm: 2.0.0
+os: linux
```

Comment:
Would you mind sharing what kind of issues you've experienced?

[We've been running such a setup](https://github.com/Homebrew/brew/blob/eec323aed81c14db6fd519aa734394ffe1071db8/.travis.yml#L3-L16) before the core/formula split in the legacy repository and while Travis too frequently had some issues, I'm not aware of any issues that could have been traced back to how we had setup this testing matrix.